components/common/Button/SubmitButton.tsx

```
@@ -0,0 +1,46 @@
+import { ReactNode } from "react";
+import { Button } from "./Button";
+import { FiCheck } from "react-icons/fi";
+import { LoadingIndicator } from "@/components/common/LoadingIndicator";
+
+interface SubmitButtonProps {
+	onClick?: (...args: any[]) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)